src/chain/polkadot/types/staking/calls.ts

```diff
@@ -1,5 +1,329 @@
 import {sts, Block, Bytes, Option, Result, CallType, RuntimeCtx} from '../support'
 import * as v0 from '../v0'
+import * as v28 from '../v28'
+import * as v9110 from '../v9110'
+import * as v9430 from '../v9430'
+
+export const bond =  {
+    name: 'Staking.bond',
+    /**
+     *  Take the origin account as a stash and lock up `value` of its balance. `controller` will
+     *  be the account that controls it.
+     * 
+     *  `value` must be more than the `minimum_balance` specified by `T::Currency`.
+     * 
+     *  The dispatch origin for this call must be _Signed_ by the stash account.
+     * 
+     *  Emits `Bonded`.
+     * 
+     *  # <weight>
+     *  - Independent of the arguments. Moderate complexity.
+     *  - O(1).
+     *  - Three extra DB entries.
+     * 
+     *  NOTE: Two of the storage writes (`Self::bonded`, `Self::payee`) are _never_ cleaned
+     *  unless the `origin` falls below _existential deposit_ and gets removed as dust.
+     *  ------------------
+     *  Base Weight: 67.87 µs
```

Comment:
Microseconds with accuracy up to the second digit past the decimal point? You can delete these rows (6 in total) if we aren't sure whether the numbers are accurate or if we don't really need that info.

Reply:
This is auto-generated code so even if we delete it now, the next time we generate the metadata for polkadot for example with the crowdloan pallet it will be there again.